src/Storages/StorageS3.cpp

```diff
@@ -1217,6 +1217,18 @@ void StorageS3::truncate(const ASTPtr & /* query */, const StorageMetadataPtr &,
 }
 
 
+StorageS3::Configuration StorageS3::updateConfiguration(ContextPtr local_context, const StorageS3::Configuration & configuration)
+{
+    StorageS3::Configuration new_configuration(configuration);
+    updateS3Configuration(local_context, new_configuration);
+    return new_configuration;
+}
+
+void StorageS3::updateConfiguration(ContextPtr local_context, StorageS3::Configuration & configuration)
+{
+    updateS3Configuration(local_context, configuration);
```

Comment:
why not just rename updateS3Configuration -> updateConfiguration ?

Reply:
OK